### DISCUSSION TITLE
why reject my student application

### DISCUSSION THREAD

**Discussion by chenmol** [community/community#50800] [GitHub Education]
I have verified the school email address and submitted information including the school admission letter, personal campus card, school transcript, etc., but my application is still rejected. Why?

**Comment by chenmol** [comment]
The rejection reason I get is this , but how can i fix it ? I have filled my full name and bio but still rejected.

> What happened?
> The item you uploaded is insufficient to demonstrate your current academic status.
> Please see suggestions.
> Suggested proof to expedite your approval:
> 
> You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

**Comment by samus-aran** [comment]
Hi @chenmol 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.
